orbbec_description/package.xml

```diff
@@ -9,6 +9,8 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <exec_depend>realsense2_gz_description</exec_depend>
```

Comment:
Will adding this dependency cause upstream issues with other repos?

Options:
1) Keep it as is
2) Comment out and make a note that this is required if `sim_gazebo` is set to true
3) Move the `realsense2_gz_description` include to the higher level URDF